nion/swift/ImageCanvasItem.py

```diff
@@ -240,13 +240,18 @@ class ScaleMarkerCanvasItemComposer(CanvasItem.BaseComposer):
     def __init__(self, canvas_item: CanvasItem.AbstractCanvasItem, layout_sizing: CanvasItem.Sizing,
                  cache: CanvasItem.ComposerCache, dimensional_calibration: typing.Optional[Calibration.Calibration],
                  info_text: str, screen_pixel_per_image_pixel: typing.Optional[float],
-                 get_font_metrics_fn: typing.Callable[[str, str], UISettings.FontMetrics]) -> None:
+                 get_font_metrics_fn: typing.Callable[[str, str], UISettings.FontMetrics],
+                 draw_on_right: bool = False,
```

Comment:
I'd like to clean up the naming here to match other naming in the code.

- `draw_on_right` should be `scale_marker_position` analogous to `legend_position` in the line plot.
- `background` should be `background_fill_color` and `None` should signify it is not drawn. Future may include `background_stroke_color` and `background_stroke_width`, similar to graphics.
- `include_info_text` should be removed for now, in my opinion. Let's put that off to a future PR.

Also, I'm open to renaming 'scale marker' to something else.